src/main/java/com/zufar/onlinestore/product/service/ProductInfoService.java

```diff
@@ -0,0 +1,10 @@
+package com.zufar.onlinestore.product.service;
+
+import com.zufar.onlinestore.product.dto.ProductInfoRequestResponseDto;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+
+public interface ProductInfoService {
+
+    Page<ProductInfoRequestResponseDto> getAllProducts(Pageable pageable);
```

Comment:
This method returns not all products, it returns some limited products
rename method name like `getProducts` please